book/src/developers/quick_setup.md

```diff
@@ -135,9 +135,9 @@ Then, in a python shell:
 ```py
 >>> from web3 import Web3
 >>> w3 = Web3(Web3.HTTPProvider("http://127.0.0.1:8545"))
->>> w3.clientVersion
+>>> w3.client_version
 'trin 0.0.1-alpha'
->>> w3.eth.blockNumber
+>>> w3.eth.block_number
 11870768
```

Comment:
```suggestion
```

It can be removed here too

Reply:
Ahh! I see, that's why I was getting an error. You want me to replace this with something else?

Reply:
Your choice you can replace it with something we use or just remove it

Reply:
Made the changes